invokeai/backend/install/invokeai_configure.py

```diff
@@ -975,8 +978,17 @@ def main() -> None:
             input("Press any key to continue...")
     except WindowTooSmallException as e:
         logger.error(str(e))
+        if backup_init_file.exists():
+            move(backup_init_file, new_init_file)
```

Comment:
Does this take care of renaming? invokeai.bak -> invokeai.yaml?

Reply:
I was considering that. But I thought it'd be nice to leave if in case the configure changed something the user doesn't like even if it was successful